custom_components/octopus_energy/services.yaml

```diff
@@ -289,4 +289,49 @@ boost_heat_pump_zone:
       selector:
         number:
           step: 0.5
+          mode: box
+
+set_heat_pump_flow_temp_config:
```

Comment:
issue: This should be added to `custom_components\octopus_energy\icons.json`, where an approriate icon is picked for the service.